src/render/dot/index.js

```diff
@@ -1,19 +1,20 @@
-const Handlebars = require("handlebars/dist/handlebars.runtime");
-const _cloneDeep = require("lodash.clonedeep");
-const options = require("../../options");
-const StateMachineModel = require("../../state-machine-model");
-const attributebuilder = require("./attributebuilder");
-const stateTransformers = require("./state-transformers");
-const transitionTransformers = require("./transition-transformers");
-const Counter = require("./counter");
-const utl = require("./utl");
+import Handlebars from "handlebars/dist/handlebars.runtime.js";
+import _cloneDeep from "lodash.clonedeep";
+import options from "../../options.js";
+import StateMachineModel from "../../state-machine-model.js";
+import attributebuilder from "./attributebuilder.js";
+import stateTransformers from "./state-transformers.js";
+import transitionTransformers from "./transition-transformers.js";
+import Counter from "./counter.js";
+import utl from "./utl.js";
+
+// eslint-disable-next-line no-unused-vars
+import _unused_dummy_to_enable_side_effects from "./dot.template.cjs";
```

Comment:
It works. But better double check what s going on here.

Reply:
at least declared the templates as having side effects in package.json so they don't get shaken out of the tree inadvertently.